app/helpers/application_helper.rb

```diff
@@ -126,7 +126,7 @@ def get_grid_alignment_css_class(alignment)
   end
 
   def get_link_target_attribute(url)
-    if url.include?(ENV.fetch('HOSTNAME'))
+    if url.include?(ENV.fetch('HOSTNAME')) || url.start_with?('/') || url.start_with?('.')
```

Comment:
Question: Do we have any existing urls that start with .? Just curious about the use case for that.